homeassistant/components/auth/__init__.py

```diff
@@ -412,4 +412,6 @@ def websocket_current_user(hass, connection, msg):
         'id': user.id,
         'name': user.name,
         'is_owner': user.is_owner,
+        'allow_change_password': any(c.auth_provider_type == 'homeassistant'
```

Comment:
I don't think that this is a good idea, it's not very extensible. What if we get other auth providers that get a UI in our frontend? I like a list of `[{auth_provider_type: 'homeassistant', auth_provider_id: 'null'}]`

Reply:
I agree, I like having the list better. I think it's inevitable that we will end up wanting this information in the frontend